eo-parser/src/test/java/org/eolang/parser/ParsingTrainTest.java

```diff
@@ -42,4 +46,31 @@ void buildsList() {
         );
     }
 
+    @Test
+    void shouldStopPipeline() {
+        final XML xml = new XMLDocument(
+            String.join(
+                "\n",
+                "<program>",
+                "  <errors/>",
+                "  <sheets/>",
+                "  <objects>",
+                "    <o abstract=\"\" line=\"1\" name=\"main\" pos=\"0\">",
+                "      <o base=\"bool\" data=\"bytes\" line=\"2\" name=\"x\" pos=\"2\">01</o>",
+                "      <o base=\"bool\" data=\"bytes\" line=\"3\" name=\"x\" pos=\"2\">00</o>",
+                "    </o>",
+                "  </objects>",
+                "</program>"
+            )
+        );
+        MatcherAssert.assertThat(
+            new Xsline(
+                new ParsingTrain()
+            ).pass(xml),
+            XhtmlMatchers.hasXPaths(
+                "/program/sheets[count(sheet)=3]"
```

Comment:
@MikhailLipanin maybe it's better to check something more meaningful? For example, we can also check that we have `critical`  error.

Reply:
done